sources/platform/actors/development/quick_start/start_web_ide.md

```diff
@@ -5,91 +5,143 @@ description: Create your first Actor using the web IDE in Apify Console.
+Once the build has been initiated, the UI will transition to the **Last build** tab, displaying the progress of the build and the Docker build log.
 
 ![Actor build](./images/actor-build.png)
 
-Notice that the UI includes four tabs - (1) Code, (2) Build, (3) Input, and (4) Run. This represents the Actor creation flow where you need first to take the source code and build it. Once you have a build, you give it input which results in an Actor run.
+:::note Actor creation flow
+
+The UI includes four tabs:
+
+- **Code**
+- **Last build**
+- **Input**
+- **Last Run**
+
+This represents the Actor creation flow, where you first build the Actor from the source code. Once the build is successful, you can provide input parameters and initiate an Actor run.
+
+:::
+
+### Run the Actor
 
-## 4. Run it
+Once the Actor is built, you can look at its input, which consists of one field - **Start URL**, the URL where the crawling starts. Below the input, you can adjust the **Run options**:
 
-Once the Actor is built, you can look at its input, which consists of one field - Start URL, the URL where the crawling starts. Below the input, you can adjust the run options consisting of build, timeout, and memory limit.
+- **Build**
+- **Timeout**
+- **Memory limit**
 
 ![Actor input](./images/actor-input.png)
 
-You can start an Actor run by pressing the **Start** button at the bottom of the page. Once the run is created, you can view its log and other details, including the **Results** which will shortly start popping up in the output tab. You can always stop the run using the **Abort** button.
+To initiate an Actor run, click the **Start** button at the bottom of the page. Once the run is created, you can monitor its progress and view the log in real-time. The **Output** tab will display the results of the Actor's execution, which will be populated as the run progresses. You can abort the run at any time using the **Abort** button.
 
 ![Actor run](./images/actor-run.png)
 
-## 5. Pull it
+### Pull the Actor
+
+To continue development locally, you can pull the Actor's source code to your local machine.
+
+:::note Prerequisites
+
+Install `apify-cli`:
+
+<Tabs>
+  <TabItem value="macOS/Linux" label="macOS/Linux">
+
+  ```bash
+  brew install apify-cli
+  ```
+
+  </TabItem>
+  <TabItem value="other platforms" label="Other platforms">
+
+  ```bash
+  npm -g install apify-cli
+  ```
+
+  </TabItem>
+</Tabs>
+
+:::
+
+To pull your Actor, you need to:
 
-For further local development, you can pull the Actor's source code to your local machine. You can get instructions for your Actor on the Actor detail page.
+1. Log in to the Apify platform
 
-![Actor pull](./images/actor-pull.png)
+    ```bash
+    apify login
+    ```
 
-To pull your Actor, you need to have `apify cli` installed. You can install it using the npm:
+2. Pull your Actor using the following command:
 
-```bash
-npm -g install apify-cli
-```
+    ```bash
+    apify pull [actor_name]
```

Comment:
Consistency question: Here we use those brackets `[]` and as an argument we use snake case but in the [CLI docs](https://docs.apify.com/cli/docs/reference#apify-pull-actorid) the arguments are only capitalized without any underscores. I found this quite inconsistent - especially when the Actor name is usually with dashes (`actor-name`). Maybe some unification should be done?

Reply:
that is a good point, I think in this case I will follow the same convention made in local development doc so `your-actor-name`